source/llassetgen/include/llassetgen/llassetgen.h

```diff
@@ -5,8 +5,7 @@
 #include "Atlas.h"
 #include "DistanceTransform.h"
 #include "FntWriter.h"
-#include "FontFinder.h"
-#include "Packing.h"
+#include "FontFinder.h"#include "Packing.h"
```

Comment:
This seems strange.

Reply:
that's definitly a merge result, i'll fix this.